mkdocs.yml

```diff
@@ -22,3 +22,4 @@ nav:
 markdown_extensions:
   - admonition
   - codehilite
+  - pymdownx.superfences
```

Comment:
This bit and `pymdown-extensions` is required to be able to write code blocks in admonitions. :)